linebot/models/responses.py

```diff
@@ -392,3 +393,24 @@ def __init__(self, available=None, genders=None, ages=None,
         self.app_types = [self.get_or_new_from_json_dict(it, AppTypeInsight) for it in app_types]
         self.subscription_periods = [self.get_or_new_from_json_dict(it, SubscriptionPeriodInsight)
                                      for it in subscription_periods]
+
+
+class InsightMessageEventResponse(Base):
+    """InsightMessageEventResponse."""
+
+    def __init__(self, overview=None, messages=None, clicks=None, **kwargs):
+        """__init__ method.
+
+        :param overview: Percentage per gender.
+        :type overview: T <= :py:class:`linebot.models.StatisticsOverview`
+        :param messages: Percentage per age group.
+        :type messages: list[T <= :py:class:`linebot.models.MessageInfo`]
+        :param clicks: Percentage per area.
+        :type clicks: list[T <= :py:class:`linebot.models.ClickInfo`]
```

Comment:
[nits]
Please fix the parameter descriptions. It seems like demographics' one.

Reply:
[imo]
In the case of `InsightDemographicResponse`, response objects have common suffix `Insight`.
If we should follow this naming convention, How about followings?
- `StatisticsOverview` -> `MessageStatistics` or `MessageOverview`
- `MessageInfo` -> `MessageInsight`, `BubbleInsight` or `MessageBubbleInsight` 
- `ClickInfo` -> `ClickInsight`

Reply:
Thanks for the good tips.